src/tests/test_scraper_base.py

```diff
@@ -7,6 +7,7 @@
 from models.initiatives import InitiativeImportState
 from platformen import Scraper
 from platformen.scraper import ScrapeException
+from tools import Geocoder
 
 
 class TestDatabase(TestCase):
```

Comment:
Same for this one. I think it makes sense to create subdirectories for each module if we start writing tests for this as well.